frontend/src/components/Template/Template.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import BlogDetail from '../../pages/BlogDetail/BlogDetail'
+import Footer from './Footer'
+import NavBar from './NavBar'
+
+const Template = () => {
+  return (
+    <div>
+        <NavBar />
+        <BlogDetail />
+        <Footer />
+    </div>
+  )
+}
```

Comment:
```suggestion
const Template = ({ children }) => {
  return (
    <div>
        <NavBar />
         { children }
        <Footer />
    </div>
  )
}
```
Make this dynamic, so everyone can use this

Then you can use it by
```
import Template ...

<Template>
   //Your content here ....
</Template>

```

Reply:
ahh okay sir, pero sa routing sir? 

![image](https://user-images.githubusercontent.com/108708225/184094648-530ecd0a-6ff0-471d-a94e-8284a3602315.png)